README.md

```diff
@@ -1,31 +1,348 @@
 # Hyperion
```

Comment:
I agree that the overview can be improved, but I disagree with your prescription for improvement. If anything such a paragraph should go at the end of the overview, and if you're putting it there, you might as well make it the opener for the Conventions section. I don't quite see the right improvement yet. We can worry about it later.